packages/server/src/lib/gost-aws.js

```diff
@@ -18,27 +19,48 @@ function getS3Client() {
                 - awslocal s3api list-objects --bucket arpa-audit-reports
         */
         console.log('------------ USING LOCALSTACK ------------');
-        const endpoint = new AWS.Endpoint(`http://${process.env.LOCALSTACK_HOSTNAME}:${process.env.EDGE_PORT || 4566}`);
-        s3 = new AWS.S3({
+        const endpoint = `http://${process.env.LOCALSTACK_HOSTNAME}:${process.env.EDGE_PORT || 4566}`;
+        console.log(`endpoint: ${endpoint}`);
+        s3 = new S3Client({
             endpoint,
-            s3ForcePathStyle: true,
+            forcePathStyle: true,
+            region: 'us-west-2',
```

Comment:
I'd [prefer](https://github.com/usdigitalresponse/usdr-gost/pull/1202#discussion_r1161909931) to avoid hard-coding regions, and defer to the [AWS SDK's documented behavior](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/setting-region.html#setting-region-order-of-precedence) for selecting the appropriate region (e.g. from `$AWS_REGION`). 

If you think not having a region specified in environments making use of LocalStack is a likely sticky point for developers, I propose setting a default in `packages/server/.env.example`, like this:
```env
AWS_REGION="${AWS_REGION:-us-west-2}"
```

Reply:
Good call, totally agree.